### DISCUSSION TITLE
How to restore work after an error

### DISCUSSION THREAD

**Discussion by rcorre** [ankitpokhrel/jira-cli#577] [Q&A] [answered]
I ran `jira issue create`, and typed a long description into the summary.
After closing it, I got:

```
jira: Received unexpected response '302 Moved Temporarily'.
```

This was a temporary network issue on my end.
Does `jira-cli` save your ticket to some temporary location that I can restore, or did I lose my work?

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @rcorre, unfortunately it doesn't save the state as of now. This is something that still needs to be worked on.

Related: https://github.com/ankitpokhrel/jira-cli/discussions/490
